spec/datadog/core/crashtracking/component_spec.rb

```diff
@@ -0,0 +1,351 @@
+require 'spec_helper'
+require 'datadog/core/crashtracking/component'
+
+require 'webrick'
+require 'fiddle'
+
+RSpec.describe Datadog::Core::Crashtracking::Component,
+  skip: Datadog::Core::Crashtracking::Component::LIBDATADOG_API_FAILURE do
+    describe '.build' do
+      let(:settings) { Datadog::Core::Configuration::Settings.new }
+      let(:agent_settings) { double('agent_settings') }
+      let(:logger) { Logger.new($stdout) }
+      let(:tags) { {} }
```

Comment:
Minor: Consider adding some fake tags here, as it's easier for an empty hash to make the tests pass accidentally on a future refactoring.